hack/run-e2e.sh

```diff
@@ -370,6 +371,15 @@ elif [ "$PROVIDER" == "gke" ]; then
         -v ${GCP_CREDENTIALS}:${GCP_CREDENTIALS}
         --env GOOGLE_APPLICATION_CREDENTIALS=${GCP_CREDENTIALS}
     )
+    # google-cloud-sdk is very large, we didn't pack it into our e2e image.
+    # instead, we use the sdk installed in CI image.
+    if [ ! -e "${GCP_SDK}/bin/gcloud" ]; then
+        echo "error: ${GCP_SDK} is not google cloud sdk, please install it here or specify correct path via GCP_SDK env"
+        exit 1
+    fi
+    docker_args+=(
+        -v ${GCP_SDK}:/google-cloud-sdk
+    )
```

Comment:
e2e image needs to run `/google-cloud-sdk/bin/gcloud config config-helper` to refresh the auth token when it expired, but the SDK is very large. we can use the SDK installed in CI image.